src/winetricks

```diff
@@ -10142,6 +10142,145 @@ load_ie8_kb2936068()
+    fi
+
+    w_download "http://download.microsoft.com/download/7/5/4/754D6601-662D-4E39-9788-6F90D8E5C097/$file1" bcff753e92ceabf31cfefaa6def146335c7cb27a50b95cd4f4658a0c3326f499
+    if [ -n "$W_UNATTENDED_SLASH_QUIET" ]; then
+        quiet="$W_UNATTENDED_SLASH_QUIET /forcerestart"
+    else
+        quiet=""
+    fi
+
+    "$WINE" "$W_CACHE/$W_PACKAGE/$file1" $quiet
+
+    case $? in
+        *) w_warn "File $file1 returned unknown error code $?"
+    esac
+
+    # Work around DLL registration bug until ierunonce/RunOnce/wineboot is fixed
+    w_try_cd "$W_SYSTEM32_DLLS"
+    for i in browseui.dll corpol.dll dxtmsft.dll dxtrans.dll ieaksie.dll ieapfltr.dll \
+             iedkcs32.dll iepeers.dll ieproxy.dll jscript.dll licmgr10.dll \
+             msdbg2.dll mshtmled.dll mstime.dll shdocvw.dll tdc.ocx urlmon.dll vbscript.dll
+    do
+        w_try_regsvr $i
+    done
+
+    w_try_cd "$W_SYSTEM64_DLLS"
+    for i in browseui.dll corpol.dll dxtmsft.dll dxtrans.dll ieaksie.dll ieapfltr.dll \
+             iedkcs32.dll iepeers.dll ieproxy.dll jscript.dll licmgr10.dll \
+             msdbg2.dll mshtmled.dll mstime.dll shdocvw.dll tdc.ocx urlmon.dll vbscript.dll
+    do
+        w_try_regsvr64 $i
+    done
+
+    # Builtin ieproxy is in system32, but ie8's lives in Program Files. Native
+    # CLSID path will get overwritten on prefix update. Setting ieproxy to
+    # native doesn't help because setupapi ignores DLL overrides. To work
+    # around this problem, copy native ieproxy to system32.
+    w_try cp -f "${W_PROGRAMS_X86_UNIX}/Internet Explorer/ieproxy.dll" "$WINEPREFIX/drive_c/windows/syswow64/"
+    w_try cp -f "${W_PROGRAMS_UNIX}/Internet Explorer/ieproxy.dll" "$WINEPREFIX/drive_c/windows/system32/"
+
+    w_override_dlls native iexplore.exe
+
+    # This is still needed. IE does not start without this regvalue
+    if w_workaround_wine_bug 25648 "Setting TabProcGrowth=0 to avoid hang"; then
+        cat > "$W_TMP"/set-tabprocgrowth.reg <<_EOF_
+REGEDIT4
+
+[HKEY_CURRENT_USER\\Software\\Microsoft\\Internet Explorer\\Main]
+"TabProcGrowth"=dword:00000000
+
+_EOF_
+        w_try_regedit "$W_TMP_WIN"\\set-tabprocgrowth.reg
+    fi
+
+    #############IE only works (a tiny bit...) setting version to win2k3!!!##########
+    #############Issue a warning with a messagebox###################################
+
+    w_try_cd "$W_TMP"
+    cat > "$W_TMP"/warning.vbs <<__EOF__
+
+    X=MsgBox( \
+    "If you want to try Internet Explorer better set windows version to Windows 2003." & vbcrlf & \
+    "For compability reasons version is now set to win7." & vbcrlf & \
+    "This will cause Internet Explorer to NOT work at all at the moment!",0+48,"Notification!")
+
+__EOF__
+
+    "$WINE" cscript.exe ./warning.vbs
+
+    ###########################End of warning#######################################
+    ################################################################################
+
+    w_set_winver win7
```

Comment:
Why? You probably should be using 'default'.

Reply:
done

Reply:
So i think i accidentaly deleted my pull request/fork

I think i have to create a new one (?)

However: I`m still stuck at the ProgramFiles issue; Either I do something wrong or winetricks returns wrong things (note i run from source tree at ~/sda2); Current code i have:

w_try cp -f "$W_CACHE/$W_PACKAGE/iexplore64.exe" "$W_PROGRAMS_UNIX/Internet Explorer/iexplore.exe"
w_try cp -f "$W_CACHE/$W_PACKAGE/iexplore32.exe" "$W_PROGRAMS_X86_UNIX/Internet Explorer/iexplore.exe"

WINE=/sda2/wine64-build/wine64 winetricks -q ie8_server2003
Executing cp -f /media/louis/4048f735-ee98-496e-b75c-40783b1eaa1b/winetricks//cache/ie8_server2003/iexplore64.exe /home/louis/.wine/dosdevices/c:/Program Files (x86)/Internet Explorer/iexplore.exe
Executing cp -f /media/louis/4048f735-ee98-496e-b75c-40783b1eaa1b/winetricks//cache/ie8_server2003/iexplore32.exe /home/louis/.wine/dosdevices/c:/Program Files (x86)/Internet Explorer/iexplore.exe

WINE=/sda2/wine64-build/wine winetricks -q ie8_server2003
Executing cp -f /media/louis/4048f735-ee98-496e-b75c-40783b1eaa1b/winetricks//cache/ie8_server2003/iexplore64.exe /home/louis/.wine/dosdevices/c:/Program Files (x86)/Internet Explorer/iexplore.exe
Executing cp -f /media/louis/4048f735-ee98-496e-b75c-40783b1eaa1b/winetricks//cache/ie8_server2003/iexplore32.exe /home/louis/.wine/dosdevices/c:/Program Files (x86)/Internet Explorer/iexplore.exe

It all expands to Program Files (x86) which is obviously wrong. What am i doing wrong/is wrong?